PowerShell/AutomationExamples/MigrationWiz/FileServerToOneDrive/ProjectImport.ps1

```diff
@@ -99,10 +99,9 @@ Log-Write -LogPath $logfile -LineValue $msg
 
 
 $azureConfiguration = New-Object -TypeName 'ManagementProxy.ManagementService.AzureConfiguration'
-$oneDriveProConfiguration = New-Object -TypeName 'ManagementProxy.ManagementService.OneDriveProConfiguration'
-
+$oneDriveProConfiguration = New-Object -TypeName 'ManagementProxy.ManagementService.SharePointOnlineConfiguration'
 $MWazureConfiguration = New-Object -TypeName           'MigrationProxy.WebApi.AzureConfiguration'
-$MWoneDriveProConfiguration = New-Object -TypeName     'MigrationProxy.WebApi.OneDriveProConfiguration'
+$MWoneDriveProConfiguration = New-Object -TypeName     'MigrationProxy.WebApi.SharePointOnlineConfiguration'
```

Comment:
I thought the script would be updated to handle both v1 and v2 (instead of just one or the other)?
I believe Anne mentioned it in the original email thread.
We might also have to think about how we make it backward compatible (for people who already use this script).